content/javascript/concepts/dates/terms/setUTCMilliseconds/setUTCMilliseconds.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: '.setUTCMilliseconds()'
+Description: 'Sets the milliseconds value of a date object based on Coordinated Universal Time (UTC).'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`.setUTCMilliseconds()`** method sets the milliseconds value of a `Date` object according to Coordinated Universal Time(UTC) and returns the updated `Date` object.
+
+## Syntax
+
+```pseudo
+myDate.setUTCMilliseconds(millisecondsValue);
+```
+
+The `.setUTCMilliseconds()` method takes a single parameter, representing the numeric value of the `milliseconds`.
+
+> **Note**: It's important to note that the `.setUTCMilliseconds()` method modifies the `Date` object in place. It returns the number of milliseconds since January 1, 1970, but it's typically not necessary to use the return value unless you specifically need it for calculations.
```

Comment:
```suggestion
> **Note**: It's important to note that the `.setUTCMilliseconds()` method modifies the `Date` object in place. It returns the number of milliseconds since January 1, 1970, but it's typically not necessary to use the return value unless it's specifically needed for calculations.
```
We avoid first- and second-person pronouns in Docs.

Reply:
Thank you, will keep this in mind while making contributions on other issues.